database/tables/workspaces.pg

```diff
@@ -1,9 +1,11 @@
 columns
+    created_at: timestamp with time zone not null default CURRENT_TIMESTAMP
     heartbeat_at: timestamp with time zone
     hostname: text
     id: bigint not null default nextval('workspaces_id_seq'::regclass)
     launched_at: timestamp with time zone
     state: enum_workspace_state not null default 'uninitialized'::enum_workspace_state
+    state_updated_at: timestamp with time zone
```

Comment:
I suggest making this a default current_timestamp as well (and maybe not null) so that it will be set to be not-null when the row is first created. As we saw with the heartbeat, this is generally more useful to have non-nulls.